SFPD - Department remodel

Marshmellow105 opened the pull request:
## About The Pull Request

Remaps the Police Department from scratch, the new department features a prisoner dropoff bay for patrol cars, a briefing area, an office space for officers to file reports, and space for TWO detectives in the detective bureau (kim kitsuragi not included).

Ports metal railings from the following PR:

- https://github.com/tgstation/tgstation/pull/81542

## Why It's Good For The Game

I think this is better than the old one.

## Testing Photographs and Procedure

<details>
<summary>Three floors and the outside carparks</summary>

![carpark](https://github.com/user-attachments/assets/ac197047-7cd8-4f7e-970b-d6da79124291)
![botright](https://github.com/user-attachments/assets/6c0fe77e-5d9a-4941-93ca-4e06364c4f95)
![botleft](https://github.com/user-attachments/assets/f8c76f64-a4df-438d-b728-256f1c3b6471)
![midleft](https://github.com/user-attachments/assets/16dbbc49-f390-4c6c-836c-0babe7acac5c)
![midright](https://github.com/user-attachments/assets/306e2743-1bc0-4121-9d46-46470ea9c1d1)
![topleft](https://github.com/user-attachments/assets/8f5ed2b1-1fc1-4d68-b6e6-a94c1dc6f1fb)
![topright](https://github.com/user-attachments/assets/d1b80b9b-1b79-48c7-a9d7-8906ded11348)


</details>

## Changelog

:cl: bomby, LemonInTheDark, Kryson
map: Budget allocated to a remodel of the SFPD
/:cl:

Marshmellow105 commented:
If this ends up TM'd while I'm at work please let me know what the feedback was, my discord is bombyyy_

XeonMations commented:
![68747470733a2f2f616666656374656461726330372e626c6f622e636f72652e77696e646f77732e6e65742f6d6462322f696d616765732f3933383835353339312f34303330363331373736352f612f686f6c64696e672f302d61646465642e706e67](https://github.com/user-attachments/assets/76d6f0bd-055f-43a2-af6a-0bdf42e43d56)
whats uh... whats holding.dmm?

AlmostLikeClockwork commented:
PEPPERSPRAY REFILL STATION RAAAAAAAAAGH

Coroneljones commented:
Looks good at a glance, only real request would probably be to port over the ATM outside of the PD, to either keep it outside or put it in the lobby, mostly for those bail (and bribe) exchanges.

XeonMations reviewed (approved):
LGTM

AllCorruptingWyrm commented:
So some highlights
![image](https://github.com/user-attachments/assets/a61a56f3-e4b6-496d-8fa3-447d8d3a2e65)

This is a very blank, expressionless hallway. I suggest you add more decals and use different floors to make rooms feel different like that.

AllCorruptingWyrm commented:
![image](https://github.com/user-attachments/assets/9de71038-809f-4c00-82e9-6569bf5ac384)
Severe lights issue on the side most room. Theres no clutter on the table either.

AllCorruptingWyrm commented:
![image](https://github.com/user-attachments/assets/51a567c5-f705-456e-9840-d36e9e889fcc)
This room /feels/ fundamentally empty. Why are the seats on the opposite end of the room? Any tables, magazine stands, etc? Consider that.

D4C-420 commented:
Going to request a console for searching prints. It's not fun (and frankly is a lot of clicking) to have to look through in the case scenario  like 50+ prints in order to even start looking for someone connected to a crime scene

Marshmellow105 commented:
![image](https://github.com/user-attachments/assets/a8405481-e057-4ac1-8582-1b2558e3758f)

Lights are present here, they were just broken in the SS (I forgot to fix all the lights before taking my screenshots)

![image](https://github.com/user-attachments/assets/e9dd9b12-a57d-4aa4-a2ba-16cc5150f95b)

I added a watercooler and potted plant. My options here are fairly limited with the available choice of flooring/objs and the nature of a 2 wide hallway. Adding any worthwhile 'nook' would impede traffic.

![image](https://github.com/user-attachments/assets/6a0c1307-7b41-4807-8f46-0ee2c4336fae)

Its a waiting area, the seats for waiting are out of the way but in the vision of the desk worker, and there is a table with the newspaper (ala medical clinic lobby magazine table) and two vendors for people waiting to use.

Marshmellow105 commented:
> Going to request a console for searching prints. It's not fun (and frankly is a lot of clicking) to have to look through in the case scenario like 50+ prints in order to even start looking for someone connected to a crime scene

I think this is out of scope for this PR. Putting a security records console would look pretty out of place and as is its base functionality would let people see the race and pictures of everyone. You could skim the computer roundstart and see a handful of nosferatu and I would imagine lots of people listed as 'vampire'

Marshmellow105 commented:
![image](https://github.com/user-attachments/assets/250fdd33-fe03-4f6b-ac40-a8bc5208a168)

with help from a good friends review I ended up on this design for the dispatch/office area. It allows passage from the alleyway to the north carpark and I think flows better.

I would appreciate a TM to see what my boys in blue think

Marshmellow105 commented:
I did not do it the normal or even efficient way, but this has no merge conflicts and is in a ready state to be testmerged

SoreYew commented:
SFPD Captains cap is a varedited head/hos security item. It is not present in this remodel. A fix of the same or a bespoke object would be nice.

Marshmellow105 commented:
> ![dreamseeker_iXnFB5YF0k](https://private-user-images.githubusercontent.com/79607211/433156599-f9d9e712-abaa-4c7b-8b9c-8135e3489de4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ1Nzc1NjYsIm5iZiI6MTc0NDU3NzI2NiwicGF0aCI6Ii83OTYwNzIxMS80MzMxNTY1OTktZjlkOWU3MTItYWJhYS00YzdiLThiOWMtODEzNWUzNDg5ZGU0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDEzVDIwNDc0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZkMzc5OTFiNmEwYmNlNjU3MTAxOGQwY2FlZTIyNTcyYjUwNWY3ZGFiNjZhOGM1MDFiZTY2NzUwNGQ1ODU0NjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.YYNPZsfHFRZdQWPOvt3FcUbmOJKZwID5Y-nznt9dLxk) ![dreamseeker_aQIOBM5JYt](https://private-user-images.githubusercontent.com/79607211/433156601-91c131d5-3322-4d85-b549-ea9fb2e8b147.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ1Nzc1NjYsIm5iZiI6MTc0NDU3NzI2NiwicGF0aCI6Ii83OTYwNzIxMS80MzMxNTY2MDEtOTFjMTMxZDUtMzMyMi00ZDg1LWI1NDktZWE5ZmIyZThiMTQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDEzVDIwNDc0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc5ZGViODc4ZWY4OTU1ODRjYjRhYjZjYmMxNDIxNzk5NzE3YzE3MDc5ZmU3ZmQ5YmZjZjhiMDI5MWQ0ZmMzOWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Cb7vWePRj3syfKFw2PEug9na5Xyydt_pp96m2GoIF1g)
> 
> This is hilarious. Please remove it.

fixed in my last push, will be in whenever the TM is bumped (@XeonMations)

Marshmellow105 commented:
I fixed my shiteating no close bitch door problem and added in the captain hat (although now its a renamed spacepol blue hat)

This is pretty much ready to go at your own discretion

XeonMations commented:
No additional issues reported.
